utest.h

```diff
@@ -1202,7 +1255,21 @@ int utest_main(int argc, const char *const argv[]) {
 
     ns = utest_ns();
     errno = 0;
+#if defined(__cplusplus)
+    try {
+      utest_state.tests[index].func(&result, utest_state.tests[index].index);
+    }
+    catch (const std::exception& err) {
```

Comment:
Also you'd need to suppress exceptions here too.

Reply:
Aye. I'll sort it.